### DISCUSSION TITLE
tutorial.py not working on carla 0.9.12

### DISCUSSION THREAD

**Discussion by newengineeeer** [carla-simulator/carla#4876] [Q&A]
when i run py -3.7 tutorial.py on the command prompt i get : 
```
destroying actors
Traceback (most recent call last):
  File "tutorial.py", line 43, in main
    world = client.get_world()
RuntimeError: time-out of 2000ms while waiting for the simulator, make sure the simulator is ready and connected to localhost:2000

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "tutorial.py", line 129, in <module>
    main()
  File "tutorial.py", line 122, in main
    camera.destroy()
UnboundLocalError: local variable 'camera' referenced before assignment
```
by using print statements and try catch structure on the part of the code saying
```
client = carla.Client('localhost', 2000)
```
i found out that the client wasn't connecting to the host which was weird since manual_control.py was working and i only added print statements after it didn't work the first 3 times i ran it, i even manually set the port to 2000 and it still didn't work. i tried re downloading carla 0.9.12 and 0.9.11 and it didn't work on both, i also pip uninstalled carla and reinstalled it with pip3 and pip and it still didn't work.

**Comment by 85151** [comment]
client = carla.Client('localhost', 2000)
client.set_timeout(20.0)
it work to me
